src/lib.rs

```diff
@@ -747,6 +747,11 @@ pub struct IterMut<'a, K: 'a, V: 'a> {
     marker: marker::PhantomData<(&'a K, &'a mut V)>,
 }
 
+/// A consuming insertion-order iterator over a `LinkedHashMap`'s entries.
+pub struct IntoIter<K, V, S> {
```

Comment:
It would be great if this could avoid having to mention `S`, but I realize that's somewhat difficult with the current setup.